Fix for this:
==> Installing to the package directory...
make[1]: Entering directory `/var/tmp/fst/src/dvd+rw-tools-6.1'
[ -d /var/tmp/fst/pkg//usr/bin ] || mkdir -p /var/tmp/fst/pkg//usr/bin
install -o root -m 0755  growisofs dvd+rw-format dvd+rw-booktype dvd+rw-mediainfo dvd-ram-control /var/tmp/fst/pkg//usr/bin
[ -d /var/tmp/fst/pkg//usr/share/man/man1 ] || mkdir -p /var/tmp/fst/pkg//usr/share/man/man1
install -o root -m 0644 growisofs.1 /var/tmp/fst/pkg//usr/share/man/man1
[ -f rpl8 ] && \
  install -o root -m 0755        rpl8 /var/tmp/fst/pkg//usr/bin
make[1]: *** [install] Error 1
make[1]: Leaving directory `/var/tmp/fst/src/dvd+rw-tools-6.1'
make: *** [install] Error 2
==> ERROR: Build Failed.  Aborting...
diff -Naur dvd+rw-tools-6.1.orig/Makefile.m4 dvd+rw-tools-6.1/Makefile.m4
--- dvd+rw-tools-6.1.orig/Makefile.m4	2006-01-20 09:08:22.000000000 +0100
+++ dvd+rw-tools-6.1/Makefile.m4	2006-02-03 19:36:06.000000000 +0100
@@ -179,14 +179,20 @@
 minus_o:=$(shell [[ `id -u` == 0 ]] && echo "-o root")
 
 install:	dvd+rw-tools
-	[[ -d $(prefix)/bin ]] || mkdir -p $(prefix)/bin
+	if [[ ! -d $(prefix)/bin ]]; then \
+		mkdir -p $(prefix)/bin; \
+	fi
 	install $(minus_o) -m $(bin_mode) $(CHAIN) $(prefix)/bin
-	[[ -d $(manprefix)/man1 ]] || mkdir -p $(manprefix)/man1
+	if [[ ! -d $(manprefix)/man1 ]]; then \
+		mkdir -p $(manprefix)/man1; \
+	fi
 	install $(minus_o) -m 0644 growisofs.1 $(manprefix)/man1
-	[[ -f rpl8 ]] && \
-	  install $(minus_o) -m $(bin_mode) rpl8 $(prefix)/bin
-	[[ -f btcflash ]] && \
-	  install $(minus_o) -m $(bin_mode) btcflash $(prefix)/bin
+	if [[ -f rpl8 ]]; then \
+		install $(minus_o) -m $(bin_mode) rpl8 $(prefix)/bin; \
+	fi
+	if [[ -f btcflash ]]; then \
+		install $(minus_o) -m $(bin_mode) btcflash $(prefix)/bin; \
+	fi
 ])
 
 # common section
